amxmodx/amx.cpp

```diff
@@ -1416,7 +1416,7 @@ static AMX_NATIVE findfunction(const char *name, const AMX_NATIVE_INFO *list, in
   int i;
 
   assert(list!=NULL);
-  for (i=0; list[i].name!=NULL && (i<number || number==-1); i++)
```

Comment:
These lists are null terminated. When is number == -1? Is this change from upstream?